components/clp-py-utils/clp_py_utils/serialization_utils.py

```diff
@@ -0,0 +1,23 @@
+import pathlib
+
+from strenum import StrEnum
+
+
+def serialize_enum(enum_value: StrEnum) -> str:
```

Comment:
accurately speaking, the argument here is a symbolic name of the enum ("enum member"): https://docs.python.org/3/library/enum.html#:~:text=An%20enumeration%3A,members%20by%20name

Shall we name the argument as `member` instead?